torchgeo/datasets/spacenet1.py

```diff
@@ -0,0 +1,215 @@
+"""Spacenet1 dataset."""
+
+import glob
+import os
+from typing import Any, Callable, Dict, List, Optional, Tuple
+
+import fiona
+import numpy as np
+import rasterio as rio
+import torch
+from affine import Affine
+from rasterio.crs import CRS
+from rasterio.features import rasterize
+from torch import Tensor
+
+from torchgeo.datasets.geo import RasterDataset
+from torchgeo.datasets.utils import (
+    check_integrity,
+    download_radiant_mlhub,
+    extract_archive,
+)
+
+
+class Spacenet1(RasterDataset):
+    """Spacenet 1: Building Detection v1 Dataset
+
+    `Spacenet 1 <https://spacenet.ai/spacenet-buildings-dataset-v1/>`_
+    is a dataset of building footprints over the city of Rio de Janeiro.
+
+    Dataset features:
+
+    * No. of images - 6940 (8 Band) + 6940 (RGB)
+    * No. of polygons - 382,534 building labels
+    * Area Coverage - 2544 sq km
+
+    Dataset format:
+
+    * Imagery - Raw 8 band Worldview-3 (GeoTIFF) & Pansharpened RGB image (GeoTIFF)
+    * Labels - GeoJSON
+
+    If you are using data from SpaceNet in a paper, please use the following citation:
+
+    * Van Etten, A., Lindenbaum, D., & Bacastow, T.M. (2018). SpaceNet: A Remote
+      Sensing Dataset and Challenge Series. ArXiv, abs/1807.01232.
```

Comment:
```suggestion
    If you are using data from SpaceNet in a paper, please cite the following paper:

    * https://arxiv.org/abs/1807.01232
```
The exact format of the citation will depend on the bibliography style, and most people will just want to ask arXiv for the bibtex output anyway.